scripts/replicate_tests_db.coffee

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env coffee
+__ = require('config').universalPath
+_ = __.require 'builders', 'utils'
+breq = require 'bluereq'
+{ exec } = require 'child_process'
+promises_ = __.require 'lib', 'promises'
+
+dbHost = require('config').db.fullHost()
+allDbsUrl = dbHost + '/_all_dbs'
+
+tests_dbs = (db)-> db.includes '-tests'
+dbUrl = (dbName)-> "#{dbHost}/#{dbName}"
+
+promises_.get allDbsUrl
```

Comment:
you actually don't need to fetch those dbs as you already have the list at hand:
```coffee
dbsBaseNames = Object.keys __.require('couch', 'list')
```

Reply:
this [list](inventaire/server/db/couch/list.coffee) do not contain `-tests` dbs. I would rather trust an explicit response from couch than a maintained hardcoded list.